Gemfile

```diff
@@ -1,11 +1,21 @@
 source 'https://rubygems.org'
 
+git_source(:github) do |repo_name|
+  repo_name = "#{repo_name}/#{repo_name}" unless repo_name.include?("/")
+  "https://github.com/#{repo_name}.git"
+end
+
 # Provides basic authentication functionality for testing parts of your engine
-gem 'solidus', github: 'solidusio/solidus'
+gem 'solidus', github: 'geminimvp/solidus', branch: 'image_overrides'
```

Comment:
I think the branch you want is `gemini_master` as that should have the overrides plus one or two other minor features we needed.